ui/api/topics.ts

```diff
@@ -15,21 +26,23 @@ export async function getTopic(
   kafkaId: string,
   topicId: string,
 ): Promise<Topic> {
-  const url = `${process.env.BACKEND_URL}/api/kafkas/${kafkaId}/topics/${topicId}?fields%5Btopics%5D=name,internal,partitions,authorizedOperations,configs`;
+  const url = `${process.env.BACKEND_URL}/api/kafkas/${kafkaId}/topics/${topicId}?${describeTopicsQuery}`;
   const res = await fetch(url, {
     headers: {
       Accept: "application/json",
     },
+    cache: "no-store"
```

Comment:
@riccardo-forina , without this, the `fetch` only calls the server the first time from `topicPage.tsx`. Further, the storage sizes and offsets are all zero in the response that gets logged. It's not clear how that's happening or where, since the server is never called.

Reply:
You did good, the default behavior of next.js is to cache all fetch responses. Adding the `cache: no-store` header opts out of this.

https://nextjs.org/docs/app/building-your-application/caching

Reply:
It's surprising though that the cached value is different from what was returned from the server.